### PR TITLE
[143] Design improvements

### PR DESCRIPTION
**Pull request by jbors**
- Palette changed for virus icon
- The color of the virus indicates the draw color
- Removed the dot behind the pen
- Header name updates

### CODE REVIEW

File: app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt [hunk 1]
```diff
@@ -175,10 +176,14 @@ class MaskActivity : AppCompatActivity(), MaskView, CaptureFlow {
     }
 
     private fun refreshBrushDrawableColor() {
+        // Retrieve the dot behind the draw icon and set to currently selected colour
         val stageBtnDrawable = binding.drawBtn.compoundDrawables[0] as LayerDrawable
```
Comment (eduardb) [inline app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt:180]: Do we still need to tint the draw button now that we tint the stages one? Seems redundant, and like a visual cacophony.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -42,10 +42,10 @@
     <string name="microscope_quality_snackbar_error">An error occurred while saving your microscope quality data. Please try again.</string>
     <string name="microscope_quality_snackbar_retry">TRY AGAIN</string>
     <!-- CaptureImage -->
-    <string name="capture_activity_title">Capture photo</string>
+    <string name="capture_activity_title">Capture a sample</string>
```
Comment (eduardb) [inline app/src/main/res/values/strings.xml:45]: I am not sure this is correct though. A sample contains multiple captured photos/images, according to our domain entities: https://github.com/theaiscope/GDD-app/blob/20117b43e48c68bc6c9c21420fe450161c4419e6/app/src/main/java/net/aiscope/gdd_app/model/SampleModels.kt

**Review by eduardb — COMMENTED** [review]
I dig the new virus icon :D But I have two questions, maybe we can make this even better.

**Comment by jbors** [comment]
> I dig the new virus icon :D But I have two questions, maybe we can make this even better.

Dunno man, this is how it came from Irina & Lorenzo. There's a bunch more changes coming. Will ask them to have a look but maybe we should set up a short call to go through the new design?

**Comment by eduardb** [comment]
> > I dig the new virus icon :D But I have two questions, maybe we can make this even better.
> 
> @eduardb Dunno man, this is how it came from Irina & Lorenzo. There's a bunch more changes coming. Will ask them to have a look but maybe we should set up a short call to go through the new design?

I see. I would prefer some async form of communication though, if possible, like Slack.

**Comment by IriOva** [comment]
> > > I dig the new virus icon :D But I have two questions, maybe we can make this even better.
> > 
> > 
> > @eduardb Dunno man, this is how it came from Irina & Lorenzo. There's a bunch more changes coming. Will ask them to have a look but maybe we should set up a short call to go through the new design?
> 
> I see. I would prefer some async form of communication though, if possible, like Slack.

First time I comment something here :). Can we talk via Slack?

**Comment by eduardb** [comment]
@IriOva sure, I have Slack open pretty much all the time, just ping me whenever works best for you.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (jbors) [reply app/src/main/res/values/strings.xml]: 'image' now

File: app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt (hunk 1 above)
Comment (jbors) [reply app/src/main/java/net/aiscope/gdd_app/ui/mask/MaskActivity.kt]: Dot is gone now

**Pull request merged**